mobile/modules/users/views/password/reset-password.client.view.html

```diff
@@ -1,4 +1,4 @@
-<form class="signin form-horizontal"
+<form class="user-login user-login--reset-password form-horizontal"
```

Comment:
probably we are not  going to use bootstrap on mobile. could we do that without special bootstrap classes?